src/shared/seccomp-util.c

```diff
@@ -465,6 +465,7 @@ const SyscallFilterSet syscall_filter_sets[_SYSCALL_FILTER_SET_MAX] = {
                 "unlinkat\0"
                 "utime\0"
                 "utimensat\0"
+                "utimensat_time64\0"
```

Comment:
What about the other unprivileged time64 system calls? Such as futex_time64, clock_gettime64.

Reply:
Indeed. Update with all time64 variants now.

Reply:
these are 32bit specific, right?

hmm, i figure we might miss some more i386 syscalls, since noone of us ever runs "systemd-analyze syscall-filters" on any host that's not x86-64, and we need to add these syscalls manually...